packages/synapse-react-client/src/utils/hooks/useDetectSSOCode.ts

```diff
@@ -33,6 +32,8 @@ export type UseDetectSSOCodeOptions = {
     resp: TwoFactorAuthErrorResponse,
     encodedTwoFaResetToken: string,
   ) => void
+  hasInitializedSession?: boolean
```

Comment:
Since this hook won't do anything unless this is true, and the default is false, consider one of the following:
* Making `hasInitializedSession` required
* Making `hasInitializedSession` default true
* Keeping the default `false`, but flipping the variable to represent the inverse